quay_search.py

```diff
@@ -0,0 +1,131 @@
+#! /home/wolffj/miniconda2/bin/python
```

Comment:
This still needs to be removed. It should be `#!/usr/bin/env python` not python2